modules/media/js/editFormIndex.js

```diff
@@ -1,2 +1,2 @@
scid}),React.createElement(SelectElement,{name:"visitLabel",label:"Visit Label",options:this.state.Data.visits,onUserInput:this.setFormData,ref:"visitLabel",required:!0,disabled:!0,value:this.state.mediaData.visitLabel}),React.createElement(SelectElement,{name:"forSite",label:"Site",options:this.state.Data.sites,onUserInput:this.setFormData,ref:"forSite",disabled:!0,value:this.state.mediaData.forSite}),React.createElement(SelectElement,{name:"instrument",label:"Instrument",options:this.state.Data.instruments,onUserInput:this.setFormData,ref:"instrument",disabled:!0,value:this.state.mediaData.instrument}),React.createElement(DateElement,{name:"dateTaken",label:"Date of Administration",minYear:"2000",maxYear:"2017",onUserInput:this.setFormData,ref:"dateTaken",value:this.state.formData.dateTaken}),React.createElement(TextareaElement,{name:"comments",label:"Comments",onUserInput:this.setFormData,ref:"comments",value:this.state.formData.comments}),React.createElement(FileElement,{name:"file",id:"mediaEditEl",onUserInput:this.setFormData,required:!0,disabled:!0,ref:"file",label:"Uploaded file",value:this.state.mediaData.fileName}),React.createElement(SelectElement,{name:"hideFile",label:"Hide File",emptyOption:!1,options:["No","Yes"],onUserInput:this.setFormData,ref:"hideFile",value:this.state.formData.hideFile}),React.createElement(ButtonElement,{label:"Update File"})))}},{key:"handleSubmit",value:function(e){e.preventDefault();var self=this,myFormData=this.state.formData;$("#mediaEditEl").hide(),$("#file-progress").removeClass("hide"),$.ajax({type:"POST",url:self.props.action,data:JSON.stringify(myFormData),cache:!1,contentType:!1,processData:!1,xhr:function xhr(){var xhr=new window.XMLHttpRequest;return xhr.upload.addEventListener("progress",function(evt){if(evt.lengthComputable){var progressbar=$("#progressbar"),progresslabel=$("#progresslabel"),percent=Math.round(evt.loaded/evt.total*100);$(progressbar).width(percent+"%"),$(progresslabel).html(percent+"%"),progressbar.attr("aria-valuenow",percent)}},!1),xhr},success:function(data){$("#file-progress").addClass("hide"),self.setState({uploadResult:"success"}),self.showAlertMessage()},error:function(err){console.error(err),self.setState({uploadResult:"error"}),self.showAlertMessage()}})}},{key:"setFormData",value:function(formElement,value){var formData=this.state.formData;""===value?formData[formElement]=null:formData[formElement]=value,this.setState({formData:formData})}},{key:"showAlertMessage",value:function(){var self=this;if(null!==this.refs["alert-message"]){var alertMsg=this.refs["alert-message"];$(alertMsg).fadeTo(2e3,500).delay(3e3).slideUp(500,function(){self.setState({uploadResult:null})})}}}]),MediaEditForm}(React.Component);MediaEditForm.propTypes={DataURL:React.PropTypes.string.isRequired,action:React.PropTypes.string.isRequired};var RMediaEditForm=React.createFactory(MediaEditForm);window.MediaEditForm=MediaEditForm,window.RMediaEditForm=RMediaEditForm,exports.default=MediaEditForm}});
```

Comment:
jsx/editFormIndex.js was deleted, so this should be too

Reply:
done!